test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTagHelpersTest.cs

```diff
@@ -47,6 +47,8 @@ public class MvcTagHelpersTest
         [InlineData("Link", null)]
         // Testing the ScriptTagHelper
         [InlineData("Script", null)]
+        // Testing the ImageTagHelper
+        [InlineData("Image", null)]
```

Comment:
Odd that MvcTagHelpersWebSite.MvcTagHelper_Home.Image.html shows up here as a binary file.  Anything unusual about it on your end?  (Content looks fine when I click on View.)